data/types/calculator/calculator.go

```diff
@@ -165,4 +178,11 @@ func (c *Calculator) Normalize(normalizer data.Normalizer) {
 	if normalizer.Origin() == structure.OriginExternal {
 		c.Units = dataBloodGlucose.NormalizeUnits(c.Units)
 	}
+
+	if c.InputTime != nil {
+		c.InputTime.Normalize(normalizer.WithReference("inputTime"))
+	}
```

Comment:
The if with always be evaluated to true
```suggestion
	c.InputTime.Normalize(normalizer.WithReference("inputTime"))
```

Reply:
that's true